src/controller/auth.js

```diff
@@ -0,0 +1,19 @@
+const user = require('../models').user;
+
+module.exports = {
+  create(req, res) {
+    req.body = JSON.parse(JSON.stringify(req.body));
+    return user
+      .create({
+        username: req.body.userName,
+        firstname: req.body.firstName,
+        lastname: req.body.lastName,
+        email: req.body.email,
+        bio: req.body.hasOwnProperty('bio') ? req.body.bio : null,
+        image: req.body.hasOwnProperty('image') ? req.body.image : null,
+        password: req.body.password,
```

Comment:
You could hash the password before saving to the database. That way, it is not stored as plain text

Reply:
Yes, I will do that now

Reply:
Also pending